tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -436,153 +438,94 @@ as other GIS and imaging processing software like QGIS and ENVI do.
+Next, we will create a proper raster using the `b9` matrix. The raster 
 format will allow us to define and manage:
 
 * Image stretch
 * Coordinate reference system & spatial reference
 * Resolution
+* and other raster attributes...
 
 It will also account for the orientation issue discussed above.
  
 To create a raster in R, we need a few pieces of information, including: 
 
 * The coordinate reference system (CRS)
-* The location of the first pixel (located in the lower left hand corner of 
-the raster). 
-* The resolution or size of each pixel in the data. 
-
-First, let's grab the spatial information that we need from the HDF5 file. 
-The CRS and associated information that is needed is stored in the `map info`
-dataset. 
-
-The map info string looks something like this:
-
-<br>
-`"UTM,1.000,1.000,256521.000,4112571.000,1.000000e+000,`
-`1.000000e+000,11,North,WGS-84,units=Meters" `. 
-</br>
-
-Notice that this information is separated by commas. We can use the 
-`strsplit` command in R to extract each element into a vector. The elements are 
-position 4 and 5 represent the lower left hand corner of the raster. We need 
-this information to define the raster's extent.
-
-```{r read-map-info}
-
-# Populate the raster image extent value. 
-# get the map info, split out elements
-mapInfo<-h5read(f,"map info")
-
-# Extract each element of the map info information 
-# so we can extract the lower left hand corner coordinates.
-mapInfo<-unlist(strsplit(mapInfo, ","))
-
-# view the attributes in the map dataset
-mapInfo
-
-```
-
-Next we define the extents of our raster. The extents will be used to calculate 
-the raster's resolution. The lower left hand corner is located at mapInfo[4:5]. 
-We can define the final raster dataset extent by adding the number of rows to 
-the Y lower left hand corner coordinate and the number of columns in the 
-`Reflectance` dataset to the X lower left hand corner coordinate. 
+* The spatial extent of the image 
 
-<figure>
-		<a href="{{ site.baseurl }}/images/hyperspectral/sat_image_lat_lon.png">
-		<img src="{{ site.baseurl }}/images/hyperspectral/sat_image_lat_lon.png"></a>
-		<figcaption>The extent of a raster represents the spatial location of each 
-		corner. The coordinate units will be determined by the spatial projection/
-		coordinate reference system that the data are in. Source: National Ecological
-		Observatory Network (NEON) </figcaption>
-</figure>
-
-<a href="{{ site.baseurl }}/raster-data-series" target="_blank"> Learn more about raster attributes including extent, and coordinate reference systems here.</a>
 
 ### Define Raster CRS
 
-We have defined the extent of our raster but we still need to define the Coordinate 
-reference system (`CRS`) of the raster. To do that, we can first grab the CRS
-string from the HDF5 attributes. Then we can assign that CRS to the raster object.
+First, we need to define the Coordinate reference system (`CRS`) of the raster. 
```

Comment:
"coordinate reference system (CRS)" since there is not direct reference to code at this point.

Reply:
yes good point. Corrected here.